src/components/Button/index.js

```diff
@@ -55,7 +43,19 @@ const StyledButton = styled(({
   pill,
   onClick,
 }) => {
-  const btnClasses = outline ? outlineBtnClasses[color] : solidBtnClasses[color]
+
+  const colorByBackground = {
+    white: 'dark-gray',
+  }
+
+  const btnClasses = `
+    b--${color}
+    ${outline
+      ? `bg-transparent ${color}` 
+      : `bg-${color} ${colorByBackground[color] || 'white'}`
+    }
```

Comment:
These 5 lines replace the need for all the `solidBtnClasses` and `outlineBtnClasses` that were there before; the only thing it is missing is hover colors but those are being replaced with a box shadow hover soon so I just removed them for now.